src/main/resources/db/postgres/defaultDS/V1.0_002__legger_til_status.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE FORESPOERSEL
+ADD SAK_STATUS VARCHAR(100) default 'UNDER_BEHANDLING' NOT NULL;
```

Comment:
Denne oppdatererer vel ikke eksisterende rader? Men er kanskje ikke noe poeng for vi har vel bare sånn 2 forespørsler i dev 🤷

Reply:
Eg trudde ikkje det først, men eg fant kode i k9-sak som gjorde det slik, så no er eg 50% sikker på at dette fungerer

Reply:
https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.postgresql.org/docs/current/ddl-alter.html&ved=2ahUKEwjdpOWkgriGAxVECBAIHSmHD6gQFnoECA0QBQ&usg=AOvVaw0cY1oVTQ25-py_pzeK6v64

Reply:
To add a column, use a command like: ALTER TABLE products ADD COLUMN description text; The new column is initially filled with whatever default value is given (null if you don't specify a DEFAULT clause).